.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,19 @@
+name: Docs
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch: {}
+
+jobs:
+  release-docs:
+    permissions:
+      contents: write
+      pages: write
+      id-token: write
+    secrets: inherit
+    uses: ./.github/workflows/reusable-publish-docs.yml
+    with:
+      version: main
```

Comment:
Just found that version here is hardcoded. I need to check how it use here released version instead of "main".

Reply:
Looking at `reusable-publish-docs.yml`, it seems like it should be the actual version alias, not a git ref? It looks like it is dropping the files into there in S3 and using it for the version label 🧐 

https://github.com/aws-powertools/powertools-lambda-java/blob/910085907687a565d04fc32303e1159963671bac/.github/workflows/reusable-publish-docs.yml#L79-L86